migrations/versions/56ce9dfdbf4f_.py

```diff
@@ -0,0 +1,45 @@
+"""empty message
+
+Revision ID: 56ce9dfdbf4f
+Revises: edaa4f9c6c21
+Create Date: 2020-12-10 09:32:22.097537
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '56ce9dfdbf4f'
+down_revision = 'edaa4f9c6c21'
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+import app
+import app.extensions
+
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('organization', schema=None) as batch_op:
```

Comment:
This update will need to be redone after the created, viewed, and camel case attribute names are fixed